code/game/objects/items/granters/crafting/rebarxbowsyndie.dm

```diff
@@ -0,0 +1,13 @@
+/obj/item/book/granter/crafting_recipe/dusting/rebarxbowsyndie_ammo
+	name = "SYNDICATE REBAR CROSSBOW OWNERS MANUAL"
+	desc = "This book will self destruct after reading."
+	crafting_recipe_types = list(
+		/datum/crafting_recipe/rebarsyndie
+	)
+	uses = 1
+	remarks = list(
+		"AIM FOR THE LEGS TO CRIPPLE YOUR FOES",
+		"USE A ROD AND WIRECUTTERS TO MAKE BETTER AMMO",
+		"BE AWARE OF THE SCOPE'S BLIND SPOTS",
+		"THIS BOOK WILL NOW SELF DESTRUCT."
```

Comment:
Does it actually self destruct? If not you wanna do that?

Reply:
It dusts itself after being read like other granter books do.